client/client.go

```diff
@@ -145,6 +158,37 @@ func New(ctx context.Context, address string, opts ...ClientOpt) (*Client, error
 		gopts = append(gopts, grpc.WithStreamInterceptor(grpc_middleware.ChainStreamClient(stream...)))
 	}
 
+	kaTimeDuration := defaultClientKeepaliveTime
+	if timeEnv := os.Getenv("OKTETO_KEEPALIVE_CLIENT_TIME_MS"); timeEnv != "" {
+		timeDuration, err := time.ParseDuration(fmt.Sprintf("%sms", timeEnv))
+		if err != nil {
+			kaTimeDuration = defaultClientKeepaliveTime
+		}
+
+		kaTimeDuration = timeDuration
+	}
+
+	kaTimeoutDuration := defaultClientKeepaliveTimeout
+	if timeoutEnv := os.Getenv("OKTETO_KEEPALIVE_CLIENT_TIMEOUT_MS"); timeoutEnv != "" {
+		timeoutDuration, err := time.ParseDuration(fmt.Sprintf("%sms", timeoutEnv))
+		if err != nil {
+			kaTimeoutDuration = defaultClientKeepaliveTimeout
+		}
+
+		kaTimeoutDuration = timeoutDuration
+	}
+
+	var kaPermitWithoutStreamDuration bool
```

Comment:
I think we should default to true here

Reply:
for that statement I would say that there are two parameters from the client and server that come into play: time and timeout (for both client and server). 
- Time: to send pings every X if there is no activity
- Timeout: to wait X for ping ack before considering the connection dead.